sechub-cli/src/daimler.com/sechub/util/zip.go

```diff
@@ -102,10 +108,32 @@ func zipOneFolderRecursively(zipWriter *zip.Writer, folder string, zContext *zip
 		}
 		relPathFromFolder := filepath.Clean(strings.TrimPrefix(fileAbs, folderAbs)) /* e.g. "/sub1"*/
 
+		// tribute to Windows...
+		relPathFromFolder = ConvertBackslashPath(relPathFromFolder)
+
+		/* Only accept source code files */
+		isSourceCode := false
+		for _, srcPattern := range zContext.config.SourceCodePatterns {
+			if strings.HasSuffix(relPathFromFolder, srcPattern) {
+				if zContext.config.Debug {
+					fmt.Printf("DEBUG: %q matches %q -> is source code\n", relPathFromFolder, srcPattern)
+				}
+				isSourceCode = true
+			}
+		}
+		if !isSourceCode { // no matches above -> ignore file
+			if zContext.config.Debug {
+				fmt.Printf("DEBUG: %q no match with source code patterns -> skip\n", relPathFromFolder)
+			}
+			return nil
+		}
+
 		/* Filter excludes */
 		for _, excludePattern := range zContext.config.Excludes {
 			if Filepathmatch(relPathFromFolder, excludePattern) {
-				//log.Printf("Excluded: %s because of pattern:'%s'", relPathFromFolder, excludePattern)
+				if zContext.config.Debug {
+					fmt.Printf("DEBUG: %q matches exclude pattern %q -> skip\n", relPathFromFolder, excludePattern)
```

Comment:
It was because LogDebug was not visible/unknown in the "util" package; it also uses the type "Context" which is unknown.
Moved the log functionality to `util` and changed it to accept just a boolean flag to indicate if debugging is active.